### PR TITLE
doc: add comms plan for aquarium

### PR DESCRIPTION
**Pull request by asettle**
Signed-off-by: Alexandra Settle <asettle@suse.com>

### CODE REVIEW

File: doc/project-plan/communication-plan.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+   * - Communication Type
+     - Description
+     - Frequency
+     - Format
+     - Distribution
+     - Owner
+   * - Weekly check-ins
+     -
+         - Weekly meeting (x3 per timezone) to discuss Aquarium.
+         - Meeting minutes that are regularly sent to the mailing list.
```
Comment (ml8mr) [inline doc/project-plan/communication-plan.rst:34]: What mailing list are we going to use for these minutes?

Comment (asettle) [reply doc/project-plan/communication-plan.rst:34]: So, I've left this intentionally blank. Currently, we're sending to ceph@suse.de but we really should have our own public ML. @l-mb is graciously looking into this.

**Review by jecluis — APPROVED** [review]

File: doc/project-plan/communication-plan.rst (hunk 1 above)
Comment (asettle) [reply doc/project-plan/communication-plan.rst:34]: Consider this a "to be updated" thing. We need to go through SUSE infra for a ML. I've added an issue for myself here: https://github.com/aquarist-labs/aquarium/issues/199

File: doc/project-plan/public-release-plan.rst [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+##########
+
+Recently, in my position as a storage engineer at SUSE I was given an opportunity
+to work on a new opinionated storage appliance called Aquarium [LINK].
+
+Aquarium is...
+
+What we are trying to achieve is...
+
+Alex is my best friend because...
```
Comment (l-mb) [inline doc/project-plan/public-release-plan.rst:148]: This seems unlikely

**Review by l-mb — APPROVED** [review]
LGTM! Thanks for writing this up!

File: doc/project-plan/public-release-plan.rst (hunk 2 above)
Comment (asettle) [reply doc/project-plan/public-release-plan.rst:148]: Excuse you

File: doc/project-plan/communication-plan.rst (hunk 1 above)
Comment (ml8mr) [reply doc/project-plan/communication-plan.rst:34]: Sounds good. I figured you were already working on a solution here. ;)

**Review by ml8mr — COMMENTED** [review]
LGTM

**Pull request merged**